public/video-ui/src/util/replaceEmptyStrings.js

```diff
@@ -0,0 +1,9 @@
+export function replaceEmptyStrings(data) {
```

Comment:
minor: `nullifyEmptyStrings` may be more explicit as we always replace with `null`